### ISSUE TITLE
.mat model not loading

### ISSUE THREAD

**Issue by HettieC** [LCSB-BioCore/COBREXA.jl#691]
```
using COBREXA
download("https://github.com/SysBioChalmers/ecModels/raw/main/ecYeastGEM/model/ecYeastGEM.mat","ecYeastGEM.mat")
model = load_model(StandardModel, "ecYeastGEM.mat")
```
Gives the following error:
```
julia> model = load_model(StandardModel, "ecYeastGEM.mat")
ERROR: MethodError: Cannot `convert` an object of type Float64 to an object of type String
Closest candidates are:
  convert(::Type{String}, ::String) at essentials.jl:218
  convert(::Type{T}, ::T) where T<:AbstractString at strings/basic.jl:231
  convert(::Type{T}, ::AbstractString) where T<:AbstractString at strings/basic.jl:232
  ...
Stacktrace:
 [1] convert(#unused#::Type{Union{Nothing, String}}, x::Float64)
   @ Base ./some.jl:36
 [2] Metabolite(id::String; name::String, formula::String, charge::Int64, compartment::Float64, notes::Dict{String, Vector{String}}, annotations::Dict{String, Vector{String}})
   @ COBREXA ~/.julia/packages/COBREXA/i07GX/src/base/types/Metabolite.jl:32
 [3] convert(#unused#::Type{StandardModel}, model::MATModel)
   @ COBREXA ~/.julia/packages/COBREXA/i07GX/src/base/types/StandardModel.jl:345
 [4] load_model(type::Type{StandardModel}, file_name::String)
   @ COBREXA ~/.julia/packages/COBREXA/i07GX/src/io/io.jl:41
 [5] top-level scope
   @ REPL[3]:1
```

**Comment by exaexa** [comment]
Hi! Looks like there is something shady in the MAT file that should be String but is in fact a number. I'll check asap

**Comment by exaexa** [comment]
ok this is a problem where `metComps` in the model are actual float (!) indexes into `comps`. I think we can add a switch for that

**Comment by HettieC** [comment]
super thanks!

**Comment by HettieC** [comment]
thanks!

**Comment by exaexa** [comment]
something like this should serve as a quick workaround for now:
```julia
model = load_model("ecYeastGEM.mat")
model.mat["metComps"] = model.mat["comps"][Int.(model.mat["metComps"])]
model = convert(StandardModel, model)
```

**Comment by HettieC** [comment]
that's perfect thank you

**Comment by exaexa** [comment]
Seems fixed in #692 . Please use COBREXA from `develop` for now, I'll try to release this asap.
